pkg/catalog/types.go

```diff
@@ -77,6 +78,9 @@ type MeshCataloger interface {
 
 	// GetInboundMeshTrafficPolicy returns the inbound mesh traffic policy for the given upstream identity and services
 	GetInboundMeshTrafficPolicy(identity.ServiceIdentity, []service.MeshService) *trafficpolicy.InboundMeshTrafficPolicy
+
+	// GetRetryPolicy returns the retry policy spec for the given upstream identity and downstream service
+	GetRetryPolicy(identity.ServiceIdentity, service.MeshService) *v1alpha1.RetryPolicySpec
```

Comment:
Should not be a part of the interface as it should be a part of the inbound/outbound traffic policies APIs.